app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -80,62 +126,16 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
     initialValues = formValues;
   }
 
-  const inPersonLocationPostOfficeSearchForm = inPersonFullAddressEntryEnabled
-    ? InPersonLocationFullAddressEntryPostOfficeSearchStep
-    : InPersonLocationPostOfficeSearchStep;
-
   const inPersonSteps: FormStep[] =
     inPersonURL === undefined
       ? []
-      : ([
-          {
-            name: 'prepare',
-            form: InPersonPrepareStep,
-            title: t('in_person_proofing.headings.prepare'),
-          },
-          {
-            name: 'location',
-            form: inPersonLocationPostOfficeSearchForm,
-            title: t('in_person_proofing.headings.po_search.location'),
-          },
-          flowPath === 'hybrid' && {
-            name: 'switch_back',
-            form: InPersonSwitchBackStep,
-            title: t('in_person_proofing.headings.switch_back'),
-          },
-        ].filter(Boolean) as FormStep[]);
+      : ([prepareFormStep, locationFormStep, flowPath === 'hybrid' && hybridFormStep].filter(
+          Boolean,
+        ) as FormStep[]);
 
   const defaultSteps: FormStep[] = submissionError
-    ? (
-        [
-          {
-            name: 'review',
-            form:
-              submissionError instanceof UploadFormEntriesError
-                ? withProps({
-                    remainingSubmitAttempts: submissionError.remainingSubmitAttempts,
-                    isResultCodeInvalid: submissionError.isResultCodeInvalid,
-                    isFailedResult: submissionError.isFailedResult,
-                    isFailedDocType: submissionError.isFailedDocType,
-                    isFailedSelfie: submissionError.isFailedSelfie,
-                    isFailedSelfieLivenessOrQuality:
-                      submissionError.selfieNotLive || submissionError.selfieNotGoodQuality,
-                    captureHints: submissionError.hints,
-                    pii: submissionError.pii,
-                    failedImageFingerprints: submissionError.failed_image_fingerprints,
-                  })(ReviewIssuesStep)
-                : ReviewIssuesStep,
-            title: t('errors.doc_auth.rate_limited_heading'),
-          },
-        ] as FormStep[]
-      ).concat(inPersonSteps)
-    : ([
-        {
-          name: 'documents',
-          form: DocumentsStep,
-          title: t('doc_auth.headings.document_capture'),
-        },
-      ].filter(Boolean) as FormStep[]);
+    ? ([reviewFormStep] as FormStep[]).concat(inPersonSteps)
+    : ([documentFormStep].filter(Boolean) as FormStep[]);
```

Comment:
It's a leftover from previous code, but I don't see that this array could ever have any falsey values that we would need the `filter(Boolean)`.

```suggestion
    : ([documentFormStep] as FormStep[]);
```